worker/state/manifold.go

```diff
@@ -184,6 +184,10 @@ func (w *stateWorker) loop() error {
 	}
 }
 
+func (w *stateWorker) Report() map[string]interface{} {
```

Comment:
Another nit: export comment. I know they're all for the engine report.

Reply:
thanks for catching it.